src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1241,9 +1241,11 @@ internal enum ErrorCode
         ERR_EncReferenceToAddedMember = 7101,
         ERR_MutuallyExclusiveOptions = 7102,
         ERR_InvalidDebugInfo = 7103,
+        WRN_FilterIsConstantFalse = 7104,
```

Comment:
These are in the wrong section; they were not, as the #region title describes, introduced in C# 6. Please move it to near the bottom.

Reply:
I put them in that section only because they are bug fixes for the error code 7095 ("FilterIsConstant") introduced in C# 6. Isn't that the right approach for error code bug fixes? If not, should it go in a new section at the bottom?

Reply:
It will be easier for the software archaeologists if we mostly add to the end.

Reply:
They have been moved to the bottom.